sql_generators/usage_reporting/templates/usage_reporting_clients_daily_v1.query.sql

```diff
@@ -0,0 +1,66 @@
+{{ header }}
+WITH usage_reporting_base AS (
+  SELECT
+    submission_timestamp,
+    DATE(submission_timestamp) AS submission_date,
```

Comment:
I wonder if we should have `app_name` and/or `normalized_app_id` here as well. I'm not entirely sure the difference between them, but I see that `normalized_app_id` is present on the baseline clients_daily tables.

Reply:
It is available in the union all the app ids:
https://github.com/mozilla/bigquery-etl/pull/6858/files#diff-205ecd62049b013a01eeb8669b85657830ff61de0a2755585e66c3bfe17296d1R10

If you'd prefer to have it already in each app, then I'd suggest we add to the the view on top of the daily_v1 table.